test: Enable the GPIO tests on EHL_CRB.

yinfangw opened the pull request:
Enable testcases under tests/drivers/gpio/gpio_basic_api
To run in twister, "-X gpio_loopback" parameter is needed.

Signed-off-by: Yinfang Wang <yinfang.wang@intel.com>

enjiamai reviewed (commented):
Hi @yinfangw please fix the compliance issue, thanks.

yinfangw committed:
test: Enable the GPIO tests on EHL_CRB.

Enable testcases under tests/drivers/gpio/gpio_basic_api
To run in twister, "-X gpio_loopback" parameter is needed.

Signed-off-by: Yinfang Wang <yinfang.wang@intel.com>
